exonum/src/sandbox/sandbox.rs

```diff
@@ -243,6 +248,13 @@ impl Sandbox {
         self.inner.borrow_mut().handle_event(event);
     }
 
+    pub fn rebroadcast(&self) {
```

Comment:
Is it ok? Or maybe change this to `handle_event` or something similar with parameter of event?

Reply:
It'd keep it as is. I doubt a generic event handler will make the code more readable right now. `self.inner.borrow_mut().handle_event(...)` pattern is used only three times in the code and all of them are distinct.

Maybe we could rename it to `recv_rebroadcast()` or something like that so that it is visually different from `broadcast()` method. It's up to you.